setup.py

```diff
@@ -22,7 +22,7 @@
         "oauthenticator>=0.13.0",
         "pem>=20.1.0",
         "pycryptodome>=3.9.8",
-        "PyJWT>=1.7.1",
+        "PyJWT==1.7.1",
```

Comment:
Is what you want to accomplish really `PyJWT==1.7.1`, or `PyJWT>=1.7.1,<2`?

I don't see any situation where we want to pin the patch version.